Rakefile

```diff
@@ -0,0 +1,28 @@
+# coding: utf-8
+require 'jekyll'
+
+OPTIONS = {
+  "incremental" => true,
```

Comment:
Since you are using ruby 2.7.1, how about changing from hash rocket to the new syntax?

Reply:
Tell me about it 🙏 (I stole this from Jekyll source code itself).

Reply:
I have read a couple of changelogs for what's new in ruby 2.7 but I couldn't find what is upgrading the hash rocket syntax. Could you provide a quick example, please? :pray: